src/__mocks__/server.ts

```diff
@@ -26,6 +26,7 @@
  ******/
 
 // mock for server.run
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
better would be to introduce a new interface to be used here as mock function return type instead of `object`.